Floofbot/Services/Repository/Models/FilteredWord.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Floofbot.Services.Repository.Models
+{
+    public partial class FilteredWord
+    {
+        [Key]
+        public ulong Id { get; set; }
+        public ulong ServerId { get; set; }
+        public string Word { get; set; }
```

Comment:
ok so the wordlist is by server rather than by channel?

Reply:
Yes, filtered words is per server